content/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Course Home"
+course_title: "Linear Algebra"
+course_image: "https://ocw.mit.edu/courses/mathematics/18-06-linear-algebra-spring-2010/18-06s10.jpg"
+course_description: "This is a basic subject on matrix theory and linear algebra. Emphasis is given to topics that will be useful in other disciplines, including systems of equations, vector spaces, determinants, eigenvalues, similarity, and positive definite matrices."
+course_highlights: "This course parallels the combination of theory and applications in Professor Strang’s textbook Introduction to Linear Algebra. The course picks out four key applications in the book: Graphs and Networks; Systems of Differential Equations; Least Squares and Projections; and Fourier Series and the Fast Fourier Transform."
+instructors: ["Gilbert Strang"]
+department: "Mathematics"
+topics: ["Linear Algebra"]
+course_number: "18.06"
+term: "Spring 2010"
+level: "Undergraduate"
+features: [
+    "Video Lectures",
+    "Exams and Solutions",
+    "Captions / Transcripts",
+    "Assignments: Problem sets with solutions"
```

Comment:
are we discarding the metadata with the path to these features?

Reply:
These are just in there as a placeholder, this is not fully fleshed out yet.